src/views/organisations/Edit.vue

```diff
@@ -12,7 +12,7 @@
             <gov-heading size="m">Edit organisation</gov-heading>
             <gov-body>General details about the organisation. To be found when searching or linked from a service page.</gov-body>
 
-            <organisation-form
+            <organisation-form v-if="!loading"
```

Comment:
This isn't needed as it's wrapped in a `<template>` which already has this condition.

```suggestion
            <organisation-form
```

Reply:
Done